docs/developers/developers-setup.md

```diff
@@ -99,55 +107,81 @@ meson compile
 meson test
 ```
 
+#### Install on system and run
+
 To install Diffuse on your system (e.g. `/usr/local/`):
 
 ```sh
 meson install # requires admin privileges
+```
+
+To run Diffuse:
 
-# Run Diffuse
+```sh
 diffuse
 ```
 
-To install Diffuse on a custom directory (e.g. `~/bin/diffuse`):
+#### Install in a custom directory and run
+
+Meson allows to change the default installation directories, see
+[command-line documentation](https://mesonbuild.com/Commands.html#configure).
+
+To install Diffuse in a custom directory (e.g. `~/bin/diffuse`):
 
 ```sh
 meson install --destdir ~/bin/diffuse
+```
+
+To run Diffuse:
 
-# Run Diffuse
+```sh
+export PYTHONPATH=$HOME/bin/diffuse/usr/local/share/diffuse
 cd ~/bin/diffuse/usr/local/bin
-PYTHONPATH=$HOME/bin/diffuse/usr/local/share/diffuse ./diffuse
+./diffuse
 ```
 
-To uninstall diffuse afterwards:
+#### Uninstall
+
+To uninstall Diffuse afterwards:
 
 ```sh
 sudo ninja uninstall -C build
 sudo rm -v /usr/local/share/locale/*/LC_MESSAGES/diffuse.mo
 ```
 
-Meson allows to change the default installation directories, see
-[command-line documentation](https://mesonbuild.com/Commands.html#configure).
+### Setup on Mac OS
 
-## Setup on Mac OS
+#### Build and test
 
-Building on Mac OS is similar to building on Linux. To recap, these are
-the steps needed to build and install Diffuse manually:
+Diffuse is using Meson as its build system.
 
-```brew install meson python3 py3cairo pygobject3 gtk+3 librsvg
-  meson setup build
-  cd build
-  meson compile
-  meson test
-  meson install
+To build and test Diffuse:
+
+```sh
+meson setup build
+cd build
+meson compile
+meson test
 ```
 
-After `meson install`, the `diffuse` command can be used to launch Diffuse
-as a native Mac app that is installed into `/Applications/Diffuse.app`.
+#### Install on system and run
+
+To install Diffuse on your system (e.g. `/usr/local/`):
+
+```sh
+meson install # requires admin privileges
+```
+
+To run Diffuse:
+
+```sh
+diffuse
+```
 
-The `diffuse` command is compatible with git. To use Diffuse as git's
```

Comment:
While not macOs specific, I think this info could be good to keep, but where? I guess put it here because I spent quite some time tweaking the launcher scripts to the macOS app compatible with git.

Reply:
I kept this line, I just moved it to ./docs/users.md as it is more a documentation for the users than for the diffuse developers.